rkt/image.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"os"
+)
+
+var (
+	cmdImage = &Command{
+		Name:    "image",
+		Summary: "Inspect the image content or extract the image. See the subcommands usage for help",
+		Usage:   "SUBCOMMAND IMAGE [file]",
+		Description: `SUBCOMMAND could be "cat-manifest", "cat-file", etc. IMAGE should be a string referencing an image; either a hash, local file on disk, or URL.
```

Comment:
Actually, we probably shouldn't even talk about arguments here, right? Just leave that to the subcommand usage. So for example
Usage: "SUBCOMMAND IMAGE [args ...]"